src/app/components/dynamic-hub/dynamic-hub.component.html

```diff
@@ -41,15 +42,18 @@ <h2>
 
 
         <div class="collection" *ngIf="cards?.size > 0">
-            <h2>{{'Collections' | translate}}</h2>
-            <div class="cards">
-                <mat-card class="card" [style.borderLeftColor]="col.value.color" style="border-left: 3px solid;"
-                    [class.disabled]="!col.value.selected" (click)="getCheckbox(!col.value.selected, col.key)"
-                    *ngFor="let col of cardCollections | keyvalue">
-                    <mat-card-header>
-                        {{col.key | getCollectionDisplayName | translate}}
-                    </mat-card-header>
-                </mat-card>
+            <div class="collection-wrapper">
+                <h2>{{'Collections' | translate}}</h2>
+                <div class="cards">
+                    <mat-card class="card" [style.borderLeftColor]="col.value.color" style="border-left: 3px solid;"
```

Comment:
Can the style be moved to the `card` class?

Reply:
It is possible for the `style="border-left: 3px solid;"`
but not the `[style.borderLeftColor]="col.value.color"`